src/services/document-service.js

```diff
@@ -104,6 +104,19 @@ class DocumentService {
     return this.documentStore.getDocumentTagsMatchingText(searchString);
   }
 
+  async findDocumentsMetadata(query) {
+    const sanitizedQuery = escapeStringRegexp(query.trim());
+
+    const queryConditions = [
+      { archived: false },
+      { title: { $regex: `${sanitizedQuery}`, $options: 'i' } }
```

Comment:
Just a tiny thing: there's no need for the template string anymore :-)

Reply:
Gone!